src/test/java/com/mysql/cj/jdbc/NonRegisteringDriverTest.java

```diff
@@ -58,6 +59,14 @@ public void setAwsProtocolOnlyToFalse() {
         software.aws.rds.jdbc.mysql.Driver.setAcceptAwsProtocolOnly(false);
     }
 
+    @AfterAll
+    public static void cleanup() throws SQLException {
+        Driver registeredDriver = DriverManager.getDriver("jdbc:mysql://localhost");
+        if (registeredDriver instanceof software.aws.rds.jdbc.mysql.Driver)
+        {
+            DriverManager.deregisterDriver(registeredDriver);
+        }
+    }
```

Comment:
nit: looks like there isn't a blank line between } and the next test

Reply:
Good catch, added the new line